pages/people/query.graphql

```diff
@@ -9,17 +9,25 @@ query {
   allDepartments(first: 100) {
     name
     parent {
-      id
+      name
+    }
+    children {
+      name
+      children {
+        name
+      }
```

Comment:
What happens here if there are more than three levels?

Reply:
Yes. The current implementation is limited to three levels. If I were to allow for a flexible depth, I suspect I could not use the children (as I would have to continually add them in the query) and instead rely on the parent. I would use a while loop `while (item.parents)` and grow the list up rather than down.